src/scrollable.directive.ts

```diff
@@ -99,7 +99,7 @@ export class NguiScrollableDirective {
     // detect the current environment
     let parentElStyle = window.getComputedStyle(parentEl);
     let scrollContainerEl = parentElStyle.overflow === 'auto' ?
-      parentEl : document.body;
+      parentEl : document.scrollingElement || document.documentElement;
```

Comment:
This will work, but a question. If `document.documentElement` works for all types of browser, why do we use `document.scrollingElement`?